src/user.spec.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-env jest */
+
+import {
+    getUser,  
+    testConnection,
+    xo,
+  } from "./util";
+  
+  describe("user", () => {
+    let userId1, userId2;
+  
+    test("getAll() works", async () => {
+      expect.assertions(1);
+      const users = await xo.call("user.getAll");
+      expect(users).toBeInstanceOf(Array);
+    });
+  
+    describe(".create()", () => {
+      test("create user <email, password>", async () => {
+        expect.assertions(2);
+  
+        userId1 = await xo.call("user.create", {
```

Comment:
Create a `user` folder which includes this file and create a local file with includes user helpers.

Create a helper that creates a user and put its id in the user's Set.